modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -340,12 +340,29 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		return
 
 	if(target.key && user != target)
-		if(iscyborg(target))
+		var/mob/living/mob_occupant = target //Get mob living
```

Comment:
the argument `target` is already typecasted to `mob/living`, uneeded new var

Reply:
Removed that var <3